Add Workspace Selection guidance (#16)
w3c-cg/solid: CONTRIBUTING.md

## Work items

In general, all documents in scope of the group are welcome. Work items can be
documents — such as Technical Reports, Primers, Best Practices and Guidelines,
Use Cases and Requirements, Surveys, QA, and so forth — or software.

### Workspace Selection

Consider the following guidance to identify the appropriate workspace to use to
efficiently manage communication and collaboration within the Solid Project.

When the work in consideration:

* involves or spans over various stakeholders, specialized groups, or
  horizontal groupings, then start a Solid Project-wide endeavor
* is about current CG work items, then it should be proposed within or across
  those current work items
* is within the scope of the CG but identifies a gap in work items, then
  follow the procedure to [propose a New Work Item](#new-work-item-proposal)

> If you are unsure where the work may fit, please create an issue detailing the
> stakeholders, problem context, and technical or knowledge goals.

Author: ```suggestion
If you are unsure where the work may fit, please create an
[issue](https://github.com/solid/specification) detailing the stakeholders,
problem context, and technical or knowledge goals.
```

## Work items

In general, all documents in scope of the group are welcome. Work items can be

> documents or software - typically Technical Reports, Primers, Best Practices
> and Guidelines, Use Cases and Requirements, Surveys, QA, and so forth.

Reviewer: ```suggestion
documents — such as Technical Reports, Primers, Best Practices and Guidelines,
Use Cases and Requirements, Surveys, QA, and so forth — or software.
```

### Workspace Selection

> Consider the following guidance to identify the appropriate workspace to
> efficiently manage communication and collaboration within the Solid Project.

Reviewer: ```suggestion
Consider the following guidance to identify the appropriate workspace to use to
efficiently manage communication and collaboration within the Solid Project.
```

When the work in consideration:

> * involves or spans across various stakeholders, specialized groups, or
>   horizontal grouping, then start a Solid Project-wide endeavor.
> * is about current CG work items, then it should be proposed within or across
>   relevant work.
> * is within the scope of the CG but identifies a gap in work items, then it
>   can follow the procedure to [proposing a New Work
>   Item](#new-work-item-proposal)

Reviewer: ```suggestion
* involves or spans over various stakeholders, specialized groups, or
  horizontal groupings, then start a Solid Project-wide endeavor
* is about current CG work items, then it should be proposed within or across
  those current work items
* is within the scope of the CG but identifies a gap in work items, then
  follow the procedure to [propose a New Work Item](#new-work-item-proposal)
```

If you are unsure where the work may fit, please create an issue detailing the

> stakeholders, problem context, and technical or knowledge goals.

Reviewer: Should indicate on which repo to create that issue, preferably including a link
